specification/managedccf/data-plane/Microsoft.mCCF/preview/2023-03-03-preview/openapi.json

```diff
@@ -0,0 +1,1912 @@
+            "name": "view_history",
+            "in": "query",
+            "required": false,
+            "type": "boolean"
+          },
+          {
+            "name": "view_history_since",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int64"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/GetCommit"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/constitution": {
+      "get": {
+        "operationId": "GetConstitution",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/cose_history": {
+      "get": {
+        "operationId": "GetCoseHistory",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/StringToBase64"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/cose_recent_proposals": {
+      "get": {
+        "operationId": "GetCoseRecentProposals",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/history": {
+      "get": {
+        "operationId": "GetHistory",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/MemberIdToSignedReq"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/js_runtime_options": {
+      "get": {
+        "operationId": "GetJsRuntimeOptions",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/JSRuntimeOptions"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/jwt/issuers": {
+      "get": {
+        "operationId": "GetJwtIssuers",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/StringToJwtIssuerMetadata"
+            }
+          }
+        }
+      }
+    },
+    "/gov/kv/jwt/public_signing_key_issuer": {
+      "get": {
+        "operationId": "GetJwtPublicSigningKeyIssuer",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "string"
```

Comment:
How is this different form `/jwt/issuers`; why is there only one?

Reply:
This is a reverse mapping of key to issuer: https://microsoft.github.io/CCF/main/audit/builtin_maps.html#jwt-public-signing-key-issuer

It's unlikely to be something users want to interact with directly, but `/kv` aims to expose all governance tables systematically.